static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-nav.js

```diff
@@ -0,0 +1,158 @@
+                    top:       null
+                });
+                config.$contentBelowMobile.css('margin-top', null);
+
+                //when scroll will pass height of the header with logo
+                if (window.scrollY >= config.headerHeight) {
+                    config.$stickyNavigation.css({
+                        position:  'fixed',
+                        top:       0,
+                        width:     '100%',
+                        'z-index': '1001'
+                    });
+
+                    //also banner below nav becomes sticky
+                    config.$bannnerMobile.css({
+                        position:  'fixed',
+                        top:       config.stickyNavigationHeight,
+                        width:     '100%',
+                        'z-index': '1000'
+                    });
+                    config.$contentBelowMobile.css('margin-top', config.belowMobileMargin);
+                }
+            } else {
+                //after 500px of scrolling 'release' banner below nav
+                config.$bannnerMobile.css({
+                    position:  'absolute',
+                    top:       500
+                });
+            }
+        }
+
+        this.variants = [
+            {
+                id: 'variant',
+                test: function () {
+                    var stickyConfig = {
+                            $stickyNavigation: $('.sticky-nav-mt-test .navigation'),
+                            $stickyTopAd: $('.sticky-nav-mt-test .top-banner-ad-container'),
+                            $header: $('.sticky-nav-mt-test .l-header__inner'),
+                            $bannnerMobile: $('.top-banner-ad-container--mobile'),
+                            $contentBelowMobile: $('#maincontent')
+                        };
+
+                    $('.sticky-nav-mt-test .l-header-main').css('overflow', 'hidden');
+                    stickyConfig.stickyNavigationHeight = stickyConfig.$stickyNavigation.dim().height;
+                    stickyConfig.headerHeight = stickyConfig.$header.dim().height;
+                    stickyConfig.belowMobileMargin = stickyConfig.stickyNavigationHeight + stickyConfig.$bannnerMobile.dim().height;
+
+                    if (detect.getBreakpoint() === 'mobile') {
+                        updatePositionMobile(stickyConfig);
+
+                        mediator.on('window:scroll', _.throttle(function () {
+                            updatePositionMobile(stickyConfig);
+                        }, 10));
+                    } else {
+                        mediator.on('window:scroll', _.throttle(function () {
+                            //height of topAd needs to be recalculated because we don't know when we will get repspond from DFP
+                            stickyConfig.stickyTopAdHeight = stickyConfig.$stickyTopAd.dim().height;
+                            updatePosition(stickyConfig);
+                        }, 10));
```

Comment:
I tried with 100 but noticed that the whole movement is little delayed. I prefer to use 10 at least for now. This is also what Darren used in his stickyMPU in RHC: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/ui/sticky.js.